cmd/substreams/pack.go

```diff
@@ -42,9 +78,10 @@ func runPack(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("marshalling package: %w", err)
 	}
 
-	if err := ioutil.WriteFile(defaultFilename, cnt, 0644); err != nil {
+	outputFile := filepath.Join(outputDir, defaultFilename)
+	if err := ioutil.WriteFile(outputFile, cnt, 0644); err != nil {
 		fmt.Println("")
-		return fmt.Errorf("writing %q: %w", defaultFilename, err)
+		return fmt.Errorf("writing %w", err)
```

Comment:
The `%w` wraps another error creating a chain of errors, convention in Go is to always separate contextual message `writing` here and the wrapped error `err` with a `:` character (and `:` should only be used once in the string and for this purposes). When the full error reaches the user/developer, it easy to spot the "error chain":

```
unable to pack: writing file: <some I/O error with the path in it>
```
```suggestion
		return fmt.Errorf("writing file: %w", err)
```